pkg/cli/cli.go

```diff
@@ -479,6 +479,22 @@ func runDeleteHostsCommand(c *cobra.Command, args []string, flags *CVDRemoteFlag
 	if err != nil {
 		return err
 	}
+	// Close connections first to avoid spurious error messages later.
+	for _, host := range args {
```

Comment:
nit: can this be moved into its own function?

Reply:
Done